### ISSUE TITLE
Suggestions on home page not changing according to what videos you watch.

### ISSUE THREAD

**Issue by hmm3** [yuliskov/SmartYouTubeTV#614]
I'm using guest mode only.  I don't know if it works with logged in accounts.  Is anyone else having this issue?  It's been like this for little over a month now.  No matter what videos you watch, the suggestions on the home page almost never change, and if they change a little bit, they don't have any relation to what you have just watched.  Moreover if they do change, if you restart the app, the suggestions reset back to what it was suggesting when you first started.  

I've tried both stable and beta, and both have the issue.  I tested a whole bunch of releases going back to 310, and all of them have issues.  I have tried clearing data or cache, or deleting watch history or search history.  None of those make a difference.  

Thanks.

**Comment by yuliskov** [comment]
I'm sure this issue is duplicate for the old one.

**Comment by hmm3** [comment]
Yes.  I noticed that there's still an open thread about this issue, but it doesn't seem to get much traffic, so I wasn't sure if the problem is a widespread problem, or something uniquely wrong with my particular setup.  As stated earlier, it was working for quite awhile even after upgrading from 310 to later versions, but suddenly stopped working.  

Thanks.

**Comment by Nzzyme** [comment]
I am using a logged in account and it works perfectly fine for me. I get related suggestions when I launch the app.
